server/integration/feature-pack/src/main/resources/modules/system/layers/base/org/infinispan/objectfilter/main/module.xml

```diff
@@ -9,7 +9,6 @@
       <module name="org.infinispan.query.dsl"/>
       <module name="org.antlr.antlr-runtime"/>
       <module name="org.hibernate.hql.parser"/>
-      <module name="org.antlr.antlr-runtime"/>
```

Comment:
We cannot delete antlr, unfortunatelly. But here it was imported twice so I removed one.